docs/pages/iconography.js

```diff
@@ -1,50 +1,39 @@
 import React from 'react'
-import { Box, Flex, Text, Icon, BlockLink } from 'pcln-design-system'
 import * as icons from 'pcln-icons'
-import { PageTitle, Description, Code } from '../src/components'
+import { Box, Flex, Text } from 'pcln-design-system'
+import { Code, Description, PageTitle } from '../src/components'
 
-const iconNames = Object.keys(icons)
+const iconNames = Object.keys(icons).filter((icon) => icon !== 'Icon')
```

Comment:
This is to remove the last non-icon that was appearing at the bottom:

<img width="729" alt="Screen Shot 2020-08-06 at 9 31 10 AM" src="https://user-images.githubusercontent.com/12076625/89545504-0e81b000-d7c9-11ea-8b60-dc8f54b03e37.png">